### PR TITLE
Example: filter

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Comment by tshemsedinov** [comment]
@aratushna here is your task, please accept invitation to github org kse-ua

File: Tasks/filter.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Filter array by type name
+
+Filter =(T,t)=> {
```
Comment (aratushna) [inline Tasks/filter.js:3]: should be const or let

File: Tasks/filter.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+for (C of T) {
+x = T.indexOf(C);
```
Comment (aratushna) [inline Tasks/filter.js:5]: should be let or const

File: Tasks/filter.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+if (typeof T[x] !== t) { T.splice(x, 1) }; };
```
Comment (aratushna) [inline Tasks/filter.js:6]: missing an indent

**Comment by aratushna** [comment]
```js
const Filter =(T,t)=> {
  for (let C in T) {
    let x = T.indexOf(C);
    if (typeof T[x] !== t) { T.splice(x, 1) }; };
  return T;
}

const result = Filter([1, 2, 'three', 4, 5, 'six'], 'number');
console.log(result);
```

optimized all mentioned in the comments above

**Comment by aratushna** [comment]
```py
def Filter(T, t):
    for C in T:
        x = T.index(C)
        if str(type(T[x])) != t:
            T.pop(x)
    return T

result = Filter([1, 2, 'three', 4, 5, 'six'], "<class 'int'>")  # replaced 'number' >> "<class 'int'>"
print(result)
```
python version

File: Tasks/filter.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+// Filter array by type name
+
+Filter =(T,t)=> {
+for (C of T) {
```
Comment (aratushna) [inline Tasks/filter.js:4]: should change 'of' to 'in'

**Pull request closed without merging**